valmlord/gulpify
# feat: added del plugin to boilerpate

## Description
This pull request adds the `del` plugin to the Gulp boilerplate, enabling the deletion of files and directories as part of the build process.

## Changes
- Added `del` package to `package.json`.
- Updated `gulpfile.js` to include a new task for cleaning the build directory using `del`.

## New Gulp Tasks
- `clear`: Deletes the contents of the build directory to ensure a fresh build.

## How to Test
1. Run `npm install` to install the new dependency.
2. Execute the `gulp clear` command.
3. Verify that the contents of the build directory are deleted.

## Additional Notes
- This update aims to improve the build process by removing old files before a new build.
